anthos-bm-gcp-terraform/main.tf

```diff
@@ -175,7 +202,7 @@ module "worker_vm_hosts" {
   zone              = var.zone
   network           = var.network
   vm_names          = local.worker_vm_names
-  instance_template = module.instance_template.self_link
+  instance_template = module.instance_template_worker.self_link
```

Comment:
**Question:** What was reason for introducing a new instance template (`instance_template_worker`)?